Sterile glove tweaks

gy1ta23 opened the pull request:
🆑
tweak: Added an extra box of sterile gloves to the infirmary, and a supply crate for more.
/🆑

In my endless ambivalence, I have once again risen from my trapped, pallid state of quasi-ERP DMs with someone that definitely isn't Ozwell to grace you all with more minute tweaks. Real change and server direction is quite simply for sissies.

A full medical department of around nine or so people at the moment has one box of sterile gloves to share between all of them. There are only five pairs of latex gloves and two pairs of nitrile gloves. The math doesn't work, which is an issue when disposable gloves are necessary PPE. Two boxes is a more reasonable amount of gloves for an infirmary to have.

Tacking onto that, if those two boxes run out, which would not be surprising, you can now order more.

Additionally, sterile glove boxes hold six latex gloves instead of five now, because even numbers are better, and if I haven't attached more tweaks to this one pull request than a stimulus bill I'm not really doing my job as an American citizen.

Spookerton reviewed (changes requested):
pleez squash

SierraKomodo reviewed (dismissed):
> tweak: "Reasonable" access to sterile gloves added

Changelog is a bit misleading and also vague. It sounds like gloves are being provided to more people when it's really just more boxes of gloves for medical to use, and the inclusion of a supply box of gloves.

Commit:
Commits = Squashed

Adds sterile glove box, adds refill crate to supply system, changes sterile glove box latex count from 5 to 6
